utils/generate-command-code.py

```diff
@@ -67,14 +67,17 @@ def get_optional_desc_string(desc, field, force_uppercase=False):
     return ret.replace("\n", "\\n")
 
 
-def check_command_key_specs(cmd):
+def check_command_key_specs(command):
     if not command.key_specs:
         return
 
     assert isinstance(command.key_specs, list)
 
+    global check_command_error_counter
```

Comment:
i'd rather return an indication of the error, and have the caller count them.
seems ugly to use a global ,